Index: README.md
===================================================================
diff --git a/README.md b/README.md
deleted file mode 100644
--- a/README.md	(revision bb93d8e9ac5c330ef1ddbe26fe27e86c34007321)
+++ /dev/null	(revision bb93d8e9ac5c330ef1ddbe26fe27e86c34007321)
@@ -1,1 +0,0 @@
-# IUM-TWEB
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision bb93d8e9ac5c330ef1ddbe26fe27e86c34007321)
+++ b/.gitignore	(date 1752580531079)
@@ -1,0 +1,1 @@
+*.csv
\ No newline at end of file
